imageregistry/v1/types.go

```diff
@@ -200,6 +200,12 @@ type ImageRegistryConfigStorageS3 struct {
 	// Optional, defaults based on the Region that is provided.
 	// +optional
 	RegionEndpoint string `json:"regionEndpoint,omitempty"`
+	// chunkSizeMiB defines the size of the multipart upload chunks of the S3 API.
+	// The S3 API requires multipart upload chunks to be at least 5MB.
```

Comment:
`5MiB` to make sure it's obvious it's binary bytes

Also, need to express in prose that the value has a maximum, and that the value is an integer number of MiB